src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -54,11 +53,10 @@ public static void DownloadData_InvalidArguments_ThrowExceptions()
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null); });
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null, null); });
 
-            AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
+            AssertExtensions.Throws<ArgumentNullException>(PlatformDetection.IsFullFramework ? "path" : "address", () => { wc.DownloadDataTaskAsync((string)null); });
```

Comment:
There is no need to to the PlatformDetection.IsFullFramework check as there is already an AssertExtensions.Throws<T> that takes both, netcore expected param name and netfx one. https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/AssertExtensions.cs#L19